relay/paseo/src/genesis_config_presets.rs

```diff
@@ -125,12 +131,16 @@ fn default_parachains_host_configuration() -> HostConfiguration<polkadot_primiti
 		scheduler_params: polkadot_primitives::SchedulerParams {
 			group_rotation_frequency: 20,
 			paras_availability_period: 4,
-			lookahead: 3,
```

Comment:
this should remain 3

Reply:
Thanks @alindima! This has been fixed in: [25eb631](https://github.com/paseo-network/runtimes/pull/212/commits/25eb63172200a99fd67433400da1d120246404f0).